src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -267,6 +268,19 @@ public DashboardWebApplication(
             options.Cookie.Name = DashboardAntiForgeryCookieName;
         });
 
+        if (builder.Configuration.GetBool("ASPNETCORE_FORWARDEDHEADERS_ENABLED") ?? false)
+        {
+            builder.Services.Configure<ForwardedHeadersOptions>(options =>
+            {
+                options.ForwardedHeaders = ForwardedHeaders.XForwardedFor | ForwardedHeaders.XForwardedHost | ForwardedHeaders.XForwardedProto;
```

Comment:
What about the `Prefix`, could we use `All` or parse from the configuration? Think maybe better support from aspnetcore framework